internal/ansible/controller/controller.go

```diff
@@ -112,7 +112,7 @@ func Add(mgr manager.Manager, options Options) *controller.Controller {
 
 	u := &unstructured.Unstructured{}
 	u.SetGroupVersionKind(options.GVK)
-	err = c.Watch(&source.Kind{Type: u}, &handler.InstrumentedEnqueueRequestForObject{}, predicates...)
+	err = c.Watch(&source.Kind{Type: u}, &handler.LoggingEnqueueRequestForObject{}, predicates...)
```

Comment:
Does this need to be

```
&handler.LoggingEnqueueRequestForObject{InstrumentedEnqueueRequestForObject: libhander.InstrumentedEnqueueRequestForObject{}}
```

or will one be instantiated implicitly?

Reply:
Implicitly. If the field was a pointer type, you'd have to instantiate it.